server/src/ems/gst/ems_gstreamer_pipeline.c

```diff
@@ -171,7 +171,7 @@ on_offer_created(GstPromise *promise, GstElement *webrtcbin)
 static void
 webrtc_on_data_channel_cb(GstElement *webrtcbin, GObject *data_channel, struct ems_gstreamer_pipeline *egp)
 {
-	U_LOG_E("called!!!!");
+	U_LOG_I("called!!!!");
```

Comment:
We could have :
U_LOG_I("webrtc_on_data_channel_cb called");

Reply:
not sure why this is still in here, tbh

Reply:
fixed.